src/panels/lovelace/views/hui-sidebar-view.ts

```diff
@@ -69,7 +69,10 @@ export class SideBarView extends LitElement implements LovelaceViewElement {
       this._createCards();
     }
 
-    if (!changedProperties.has("lovelace")) {
+    if (
+      !changedProperties.has("lovelace") &&
+      !changedProperties.has("_config")
```

Comment:
This is the wrong check. Now you won't do anything unless _both_ Lovelace and config update.